x/storage/keeper/query.go

```diff
@@ -301,7 +301,7 @@ func (k Keeper) VerifyPermission(goCtx context.Context, req *types.QueryVerifyPe
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
 	operator, err := sdk.AccAddressFromHexUnsafe(req.Operator)
-	if err != nil {
+	if err != nil && err != sdk.ErrEmptyHexAddress {
```

Comment:
Does only this line of code really affect anonymous users' access to public objects?

Reply:
Here just skip the empty address check. The key point is in VerifyObject/BucketPermission